app/src/main/java/br/com/fundatec/fundatecheros/home/views/profile/view/PofileActivity.kt

```diff
@@ -0,0 +1,100 @@
+package br.com.fundatec.fundatecheros.home.views.profile.view
+
+import android.os.Bundle
+import android.util.Patterns
+import androidx.appcompat.app.AppCompatActivity
+import br.com.fundatec.fundatecheros.databinding.ActivityProfileBinding
+import com.google.android.material.snackbar.Snackbar
+
+class PofileActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityProfileBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityProfileBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        nameFocusListener()
+        emailFocusListener()
+        passwordFocusListener()
+
+
+        binding.buttonCreateNewUser.setOnClickListener { submitForm() }
+
+    }
+
+    private fun submitForm() {
+        val validName = binding.tilName.helperText == null
+        val validEmail = binding.tilEmail.helperText == null
+        val validPassword = binding.tilPassword.helperText == null
```

Comment:
O `helperText` não possui acesso ao texto que o usuário inseriu nos campos, então essas validações não estão funcionando
```suggestion
        val validName = binding.tilName.text == null
        val validEmail = binding.tilEmail.text == null
        val validPassword = binding.tilPassword.text == null
```